fix: updated sharing and collaboration feature (#2807)
gitpod-io/website: gitpod/docs/sharing-and-collaboration.md

<script context="module">
  export const prerender = true;
  import IdeToggle from "$lib/components/docs/ide-toggle.svelte";
</script>

# Collaboration & Sharing of Workspaces

There are currently two ways to share a Gitpod workspace, either by creating a [workspace snapshot](#sharing-snapshots), a "point in time" capture of a workspace state that can be then opened by another user. Or, by [sharing a running workspace](#sharing-running-workspaces) which shares access to a single running workspace to multiple users.

`youtube: TrY15zN98p8`

## Workspace Snapshots

> Snapshotting a workspace is useful when you want to create reproducible workspace state for reporting support issues, or when giving training, or presentations. You can create up to three snapshots from any workspace.

Author: Is this limitation still true: "You can create up to three snapshots from any workspace."

You can create a snapshot URL of your workspace to share your workspace files with others. Following a Gitpod snapshot URL starts a new workspace, but under the account of the user who followed that original snapshot URL.

There are a few important things to note about workspace snapshots:

> - **Snapshots are treated as workspace restarts** - All workspace state, including files changed since the original workspace start are preserved in a snapshot. Any tasks or processes typically launched via a workspace start will not run for a workspace snapshot.

Author: ```suggestion
- **Snapshots behave like workspace restarts** - All workspace state, including files changed since the original workspace start are preserved in a snapshot. Any tasks or processes typically launched via a workspace start will not run for a workspace snapshot.
```

> - **Access to secret information** - Snapshotted workspaces do not grant access to Gitpod environment variables, Gitpod authentication or Git credentials within a snapshot. However, you should use caution to ensure that any secret or private information is not stored in any files of a workspace snapshot before generating a snapshot URL.

Author: ```suggestion
- **Access to secret information** - Snapshotted workspaces do not grant access to Gitpod environment variables, Gitpod authentication or Git credentials within a snapshot. However, you should use caution that you did not store any secret or private information in any files of a workspace snapshot before generating a snapshot URL.
```

> - **Deleting and managing snapshots** - If you want to remove any created snapshot URLs, deleting the workspace directly removes any created snapshot URLs associated with that workspace.

Author: @svenefftinge / @csweichel  - Is it worth mentioning our periodic cleanup here also?

```suggestion
- **Deleting and managing snapshots** - If you want to remove any created snapshot URLs, deleting the workspace directly removes any created snapshot URLs associated with that workspace. Gitpod also periodically deletes old workspaces (typically on a 2 week cycle), unless a workspace is pinned. 
```

Reviewer: I would simplify this:
> Deleting and managing snapshots** - If you want to remove any created snapshot URLs, deleting the workspace directly removes any created snapshot URLs associated with that workspace.
 
To:
> Deleting snapshots** - Deleting a workspace deletes any existing snapshots as well.
